specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/ResourceProvider.json

```diff
@@ -1605,6 +1697,32 @@
           "x-ms-client-flatten": true
         }
       }
+    },
+    "VirtualNetworkIntegrationRequest": {
+      "description": "Virtual Network Integration request content.",
+      "required": [
+        "subnetResourceId"
+      ],
+      "type": "object",
+      "properties": {
+        "subnetResourceId": {
+          "description": "The Azure Resource ID of the subnet",
+          "type": "string",
+          "format": "arm-id"
+        }
+      }
+    },
+    "PurgedUnusedVirtualNetworkSuccessResponse": {
+      "description": "Response content upon successful deletion of an unused virtual network.",
+      "type": "string",
+      "enum": [
+        "Purged unused virtual network integration.",
+        "No unused virtual network integration."
```

Comment:
I think you should choose enum names that are shorter, easier to type, or look more like programmatic identifiers! :)